app/lib/core/services/database/database_service.dart

```diff
@@ -1,2 +1,14 @@
-// TODO: #130. Add the service which uses the database interface to provide work
-class DatabaseService {}
+import 'dart:async';
+
+import 'package:app/core/models/test_animal_transport_record.dart';
+import 'package:app/ui/widgets/atr_preview.dart';
+
+class DatabaseService {
+  // TODO: #130. Replace with actual DB functions
+  Stream<AnimalTransportRecordPreview> getUpdatedPreviews() async* {
```

Comment:
Cheap stream that runs forever. No intensive by any means but this also isn't what we want in actual implementation.